contrib/meson.build

```diff
@@ -12,6 +12,10 @@ configure_file(
   configuration: con2,
 )
 
+if get_option('tests')
+  install_data('lsan-suppressions.txt', install_dir: installed_test_datadir)
```

Comment:
for something we're effectively installing by default, I think it's time to move `lsan-suppressions.txt` out of `contrib/`

Reply:
to `data` perhaps?

Reply:
I'd say `data` or `data/tests` can both be argued.  Either is fine with me.

Reply:
Ahh, data/tests makes it very easy indeed. Fixed!